vision/body_analysis/arcface/model/arcfaceresnet100-opset8-md5.txt

```diff
@@ -0,0 +1 @@
+MD5 (resnet100.onnx) = 66074b860f905295aab5a842be57f37d
```

Comment:
File contents (small formatting fix ) should be as follows:

66074b860f905295aab5a842be57f37d  arcfaceresnet100-8.onnx

Reply:
As per our previous conversation on gitter, this file can be deprecated entirely.

Reply:
removed the file and the link to it